pythonforandroid/recipes/python2/__init__.py

```diff
@@ -117,11 +125,12 @@ def do_python_build(self, arch):
                     ' _sqlite/util.c',
                     ' _sqlite/row.c',
                     ' -DSQLITE_ENABLE_FTS4',
-                    #' -DMODULE_NAME _sqlite3',
+                    ' -D\'MODULE_NAME="_sqlite3"\'',
```

Comment:
RAN: '/usr/bin/make -j5 install HOSTPYTHON=/home/paul/.local/share/python-for-android/build/other_builds/python2-openssl-sqlite3/armeabi/python2/hostpython HOSTPGEN=/home/paul/.local/share/python-for-android/build/other_builds/python2-openssl-sqlite3/armeabi/python2/hostpython CROSS_COMPILE_TARGET=yes INSTSONAME=libpython2.7.so'

  STDOUT:
Makefile:208: *** missing separator.  Stop.